enumerate_iam/main.py

```diff
@@ -62,17 +64,24 @@ def enumerate_using_bruteforce(access_key, secret_key, session_token, region):
     """
     Attempt to brute-force common describe calls.
     """
+    global STOP_SIGNAL
     output = dict()
 
     logger = logging.getLogger()
     logger.info('Attempting common-service describe / list brute force.')
 
+    # Ignore SIGINT signals so that child processes inherit SIGINT handler
```

Comment:
`from multiprocessing.dummy import Pool as ThreadPool` is in use, there are no child processes.

Reply:
Fair point. Let's ignore that comment.